flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/CEPITCase.java

```diff
@@ -708,10 +724,32 @@ public boolean filter(Tuple2<Integer, String> rec) throws Exception {
 			}
 		});
 
-		result.writeAsText(resultPath, FileSystem.WriteMode.OVERWRITE);
+		CollectSink.VALUES.clear();
 
-		expected = "(1,a)\n(3,a)";
+		result.map(new MapFunction<Tuple2<Integer, String>, String>() {
+			@Override
+			public String map(Tuple2<Integer, String> value) throws Exception {
+				return value.toString();
```

Comment:
please do not compare as strings. This was only done for simplicity in the previous code so we don't have to re-parse the contents of the text file.

This also applies to other tests.